src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/products/ProductSyncIT.java

```diff
@@ -305,14 +306,14 @@ public void sync_withProductTypeReference_ShouldUpdateProducts() {
         final ProductDraft existingProductDraft2 = createProductDraft(PRODUCT_KEY_2_RESOURCE_PATH,
             targetProductType.toReference(), targetTaxCategory.toReference(), targetProductState.toReference(),
             targetCategoryReferencesWithIds, createRandomCategoryOrderHints(targetCategoryReferencesWithIds));
-        final Product targetProductWithKey2 = CTP_TARGET_CLIENT.execute(ProductCreateCommand.of(existingProductDraft2))
-                                                               .toCompletableFuture().join();
+        CTP_TARGET_CLIENT.execute(ProductCreateCommand.of(existingProductDraft2))
+                         .toCompletableFuture().join();
 
         final ProductDraft existingProductDraft3 = createProductDraftBuilder(PRODUCT_KEY_2_RESOURCE_PATH,
```

Comment:
because the for the other product drafts (there are already existing JSON resources for them: `product-key-1.json` and `product-key-2.json`) but since sometimes we just want to create some draft with a minor change, we don't always need a new resource for it. So I can just `createProductDraftBuilder` and override the values on the other resources to dynamically create new drafts more efficiently (without having to persist resource files).